### ISSUE TITLE
Add tests to validate developers efforts

### ISSUE THREAD

**Issue by gaand** [ga-wdi-boston/ruby-vs-js-arrays-study#1]
For both ruby and js.

**Comment by gaand** [comment]
This requires that this repo (and similar) use both the ruby and node template functionality.

**Comment by neugierige** [comment]
Currently, this study doesn't actually require developers to write any Ruby, since it is only the "vs-js" part. The Ruby part was done during lab. What would you propose? Should they to repeat the exercise they had done in lab?

**Comment by gaand** [comment]
I'm not sure on how much should be added, and nothing for this cycle.

I'm striving for more tests on more repositories, and this is so it's not forgotten.

**Comment by neugierige** [comment]
To confirm, I'm going to write tests for js, and we are not adding anything for Ruby. Is that accurate?

**Comment by gaand** [comment]
No, you do not have to write tests.  That will happen in the next cycle when someone else will handle this issue.

**Comment by Jcornmanhomonoff** [comment]
@gaand do tests still need to be written for this?

**Comment by jrhorn424** [comment]
@Jcornmanhomonoff Yes, but first: #48
